integration-cli/docker_cli_run_test.go

```diff
@@ -1353,6 +1353,17 @@ func TestRunWithCpusetCpus(t *testing.T) {
 	logDone("run - cpuset-cpus 0")
 }
 
+func TestRunWithBlkioWeight(t *testing.T) {
```

Comment:
Could you possibly start 2 containers, one with high weight and the other with low weight and make sure the high weight finished first ?